app/modules/import-export-customization/runners/import/elementor-content.php

```diff
@@ -98,25 +105,29 @@ private function import_elementor_post_type( array $posts_settings, $path, $post
 
 					if ( is_array( $import_result ) ) {
 						$result[ $import_result['status'] ][ $id ] = $import_result['result'];
+						$this->map_imported_post_id( $id, $import_result );
```

Comment:
I have copied most of the logic for parent/child handling from the `wp-import.php` file. We did have support for this for WordPress pages, but not for Elementor pages.